android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/login/LoginScreen.kt

```diff
@@ -0,0 +1,155 @@
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import co.nimblehq.ic.kmm.suv.android.R
+import co.nimblehq.ic.kmm.suv.android.ui.components.PrimaryButton
+import co.nimblehq.ic.kmm.suv.android.ui.components.PrimaryTextField
+import kotlinx.coroutines.delay
+
+@Composable
+fun LoginScreen() {
+    var email by remember { mutableStateOf("") }
+    var password by remember { mutableStateOf("") }
+    var componentsVisible by remember {  mutableStateOf(false) }
+
+    LaunchedEffect(Unit) {
+        delay(4000)
+        componentsVisible = true
+    }
+
+    Box {
+        Box(modifier = Modifier.fillMaxSize()) {
+            BackgroundImage(modifier = Modifier.matchParentSize())
+        }
+        NimbleLogo(
+            modifier = Modifier.align(Alignment.Center)
+                .wrapContentHeight()
+                .navigationBarsPadding()
+                .imePadding()
+        )
+
+        AnimatedVisibility(
+            visible = componentsVisible,
+            enter = fadeIn(),
+            exit = fadeOut(),
+            modifier = Modifier
+                .fillMaxWidth()
+                .align(Alignment.Center)
+        ) {
+            Column(
+                verticalArrangement = Arrangement.spacedBy(20.dp),
+                modifier = Modifier
+                    .fillMaxSize()
+                    .wrapContentHeight()
+                    .navigationBarsPadding()
+                    .imePadding()
+            ) {
+                PrimaryTextField(
+                    value = email,
+                    onValueChange = { email = it },
+                    placeholder = stringResource(id = R.string.login_email),
+                )
+                PrimaryTextField(
+                    value = password,
+                    onValueChange = { password = it },
+                    placeholder = stringResource(id = R.string.login_password),
+                    visualTransformation = PasswordVisualTransformation(),
+                )
+                PrimaryButton(
+                    title = stringResource(id = R.string.login_button),
+                    onClick = { /* TODO: Handle in integrate task */ },
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun BackgroundImage(modifier: Modifier) {
+    var blurState by remember { mutableStateOf(0f) }
+    val animateBlurState by animateFloatAsState(
+        targetValue = blurState,
+        tween(
+            durationMillis = 500,
+            delayMillis = 500
+        )
+    )
+    LaunchedEffect(Unit) {
+        delay(3000)
+        blurState = 20f
+    }
+    Image(
+        painter = painterResource(id = R.drawable.bg_splash),
+        contentDescription = null,
+        contentScale = ContentScale.FillWidth,
+        modifier = modifier
+            .blur(radius = animateBlurState.dp)
+    )
+}
```

Comment:
I don't see the blurState working 🤔 

- Pixel 2XL - Android 11



https://user-images.githubusercontent.com/16315358/192937871-61763358-e56e-4533-8a38-c3dbcdb2bb0e.mp4

Reply:
The `modifier.blur` does not seem to work on Android 11 and lower https://www.reddit.com/r/mAndroidDev/comments/v62ukr/blur_in_jetpack_compose_supported_only_in_android/. Flutter makes life easier 😮‍💨 

- Pixel 6 - Android 13 ✅ 


https://user-images.githubusercontent.com/16315358/192938382-c855c54a-2404-4818-bd22-42a056e5f794.mp4